enhancements/installer/nutanix-ahv-ipi.md

```diff
@@ -0,0 +1,183 @@
+
+## Motivation
+
+Red Hat and Nutanix have recently [announced](https://www.redhat.com/en/about/press-releases/red-hat-and-nutanix-announce-strategic-partnership-deliver-open-hybrid-multicloud-solutions) a strategic partnership to deliver open hybrid multicloud solutions. For OpenShift this means that it ​​becomes the preferred enterprise full-stack Kubernetes solution on Nutanix Cloud Platform with AHV.
+
+Today, OpenShift can be installed on the Nutanix AOS platform using the OpenShift platform [agnostic installer](https://github.com/nutanix/openshift/tree/main/docs/install/manual), but mutual customers have also requested a fully automated, integrated installer built using OpenShift 4's infrastructure level automation capabilities.
+This is particularly appealing for customers deploying in "hybrid cloud" scenarios where OpenShift should be deployed using common mechanisms across different clouds.
+
+### Goals
+
+The primary goal of Nutanix AHV IPI is to provide users with an easier path to running OpenShift 4 on Nutanix AOS with AHV.
+
+To achieve that goal, we will:
+
+- Enhance the installer, adding Nutanix AOS options.
+
+- Prepare Nutanix infrastructure using the [Nutanix Terraform (TF) provider](https://github.com/nutanix/terraform-provider-nutanix) for provisioning.
+
+- Integrate with the Nutanix machine API provider [to-do] and enhance requisite operators to enable cluster functionality.
+
+- Provide Nutanix AHV IPI user documentation [here](https://github.com/openshift/installer/tree/master/docs/user/nutanix/).
+
+- Provide CI artifacts required to test the Nutanix AHV IPI installer.
+
+### Non-Goals
+
+None at this time.
+
+## Proposal
+
+- Implement installer CLI prompts to build a default/minimal `install-config.yaml` for Nutanix AHV:
+
+  ```shell
+  ? SSH Public Key /Users/someone/.ssh/id_rsa.pub
+  ? Platform nutanix
+  ? Prism Central pc01.ocp.nutanix.com
+  ? Prism Central Username admin@ocp.nutanix.com
+  ? Prism Central Password [? for help] **********
+  ? Cluster (Prism Element) nx01.ocp.nutanix.com
+  ? Container default-container-70593
+  ? Subnet vm-network
+  ? Virtual IP Address for API 192.168.1.5
+  ? Virtual IP Address for Ingress 192.168.1.6
+  ? Base Domain ocp.nutanix.com 
+  ? Cluster Name ocp01
+  ? Pull Secret [? for help] ****
+  ```
+
+- Implement Nutanix specific platform installer customizations:
+  
+  **Platform**
+  
+  - prismCentral (required string): Name or IP address of the Nutanix Prism Central instance to target.
+  - prismCentralUser (required string): Username for Prism Central access (Cluster Admin).
+  - prismCentralPass (required string): Password for Prism central access.
+  - cluster (required string): Nutanix cluster (Prism Element) where the OpenShift cluster will be created.
+  - container (required string): Nutanix container name for OS disk install.
+  - subnet (required string): Nutanix subnet name (IPAM or DHCP Enabled).
```

Comment:
Would it be possible to add a sample 'platform' section of an install-config.yaml?

Reply:
Done